src/post/post.service.ts

```diff
@@ -79,6 +84,35 @@ export class PostService {
         return postAttachments;
     }
 
+    private async getMentionedAccounts(
+        object: Note | Article,
+    ): Promise<Account[]> {
+        const accounts: Account[] = [];
+        for await (const tag of object.getTags()) {
+            if (tag instanceof FedifyMention) {
+                if (!tag.href) {
+                    continue;
+                }
+
+                if (!tag.name) {
+                    continue;
+                }
+
+                const accountResult = await this.accountService.ensureByApId(
```

Comment:
This can potentially make a network request, but as far as I can see it would only be in the case of:

1. An external Note
2. Mentions an external Account

In that case we probably don't actually care about the mention, but I don't think it's harmful to collect and store it anyway